test/controllers/nonsipp/memberdetails/CheckMemberDetailsFileControllerSpec.scala

```diff
@@ -66,7 +66,7 @@ class CheckMemberDetailsFileControllerSpec extends ControllerBaseSpec {
     reset(mockAuditService)
     mockStream()
     mockSaveValidatedUpload()
-    mockValidateCSV(UploadFormatError, 0, 0L)
+    mockValidateCSV((UploadFormatError, 0, 0L))
```

Comment:
Double check please the parenthesis.

Reply:
```
  private def mockValidateCSV(result: (Upload, Int, Long)): Unit =
    when(mockMemberDetailsUploadValidator.validateCSV(any(), any(), any(), any())(any(), any()))
      .thenReturn(Future.successful(result))
```
mockValidateCSV is expecting a tuple. if you don't put these parentheses, compiler will add for you but produce warning in the log.